iml-wasm-components/iml-stratagem/src/inode_table.rs

```diff
@@ -0,0 +1,267 @@
+use bootstrap_components::bs_table::table;
+use futures::Future;
+use seed::{
+    error,
+    fetch::{FailReason, FetchObject, Request, RequestController},
+    prelude::*,
+    td, tr,
+};
+
+#[derive(serde::Deserialize, serde::Serialize, Debug, PartialEq, Clone)]
+pub struct INode {
+    counter_name: String,
+    count: u32,
+}
+
+#[derive(Default, Debug)]
+pub struct Model {
+    inodes: Option<Vec<INode>>,
```

Comment:
```suggestion
    inodes: Vec<INode>,
```

This doesn't need an option wrapper.